docs/userguide/recommenders.md

```diff
@@ -0,0 +1,118 @@
+
+### Clustering Recommenders
+
+BayBE offers a set of recommenders leveraging techniques to facilitate point selection
+via clustering:
+* **[`PAMClusteringRecommender`](baybe.recommenders.pure.nonpredictive.clustering.PAMClusteringRecommender):**
+  This recommender utilizes partitioning around medoids.
+* **[`KMeansClusteringRecommender`](baybe.recommenders.pure.nonpredictive.clustering.KMeansClusteringRecommender):**
+  This recommender implements k-means clustering.
+* **[`GaussianMixtureClusteringRecommender`](baybe.recommenders.pure.nonpredictive.clustering.GaussianMixtureClusteringRecommender):**
+  This recommender leverages Gaussian Mixture Models for clustering.
+
+### Sampling Recommenders
+
+BayBE provides two recommenders that recommend by sampling form the search space:
+* **[`RandomRecommender`](baybe.recommenders.pure.nonpredictive.sampling.RandomRecommender):**
+  This recommender offers random recommendations for all types of search spaces.
+  It is extensively used in backtesting examples, providing a valuable comparison.
+  For detailed usage examples, refer to the list
+  [here](./../../examples/Backtesting/Backtesting).
+* **[`FPSRecommender`](baybe.recommenders.pure.nonpredictive.sampling.FPSRecommender):**
+  This recommender is only applicable for discrete search spaces, and recommends points
+  based on farthest point sampling. A practical application showcasing the usage of
+  this recommender can be found
+  [here](./../../examples/Custom_Surrogates/surrogate_params).
+
+## Meta Recommenders
+
+In analogy to meta studies, meta recommenders are wrappers that operate on a sequence
+of pure recommenders and determine when to switch between them according to different
+logics. BayBE offers three distinct kinds of meta recommenders.
+
+* The
+  [`TwoPhaseMetaRecommender`](baybe.recommenders.meta.sequential.TwoPhaseMetaRecommender)
+  employs two distinct recommenders and switches between them at a certain specified
+  point, controlled by the `switch_after` attribute. This is useful e.g. if you want a
+  different recommender for the initial recommendation when there is no data yet
+  available. This simple example would recommend randomly for the first batch and switch
+  to a Bayesian recommender as soon as measurements have been ingested:
+```python
+from baybe.recommenders import (
+    TwoPhaseMetaRecommender,
+    RandomRecommender,
+    SequentialGreedyRecommender,
+)
+
+recommender = TwoPhaseMetaRecommender(
+    initial_recommender=RandomRecommender(), recommender=SequentialGreedyRecommender()
+)
+```
+
+* The **[`SequentialMetaRecommender`](baybe.recommenders.meta.sequential.SequentialMetaRecommender)**
+  introduces a simple yet versatile approach by utilizing a predefined list of
+  recommenders. By specifying the desired behavior using the `mode` attribute, it is
+  possible to flexibly determine the strategy's response when it exhausts the available
```

Comment:
Double-check if this "strategy" should be replaced by "meta recommender"

Reply:
fixed